tests/impl.cpp

```diff
@@ -6449,39 +6449,35 @@ result_t test_mm_hadd_pi32(const SSE2NEONTestImpl &impl, uint32_t i)
 result_t test_mm_hadds_epi16(const SSE2NEONTestImpl &impl, uint32_t i)
 {
     int16_t s16_max = 0x7fff;
+    int16_t s16_min = 0x8000;
```

Comment:
Append some comments for `s16_{max,min}`, such as the motivations.

Reply:
I decide to remove the variables and replace them with macro `INT16_MAX` and `INT16_MIN`.